var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -298,19 +290,33 @@ def setup_environment(self, spack_env, run_env):
         # Set Dyninst RT library path to support OSS loop resolution code
         run_env.set('DYNINSTAPI_RT_LIB', dyninst_libdir)
 
-        # Find openspeedshop library path
-        oss_libdir = find_libraries('libopenss-framework',
-                                    root=self.spec['openspeedshop'].prefix,
-                                    shared=True, recursive=True)
-        run_env.prepend_path('LD_LIBRARY_PATH',
-                             os.path.dirname(oss_libdir.joined()))
-
         run_env.set('OPENSS_RAWDATA_DIR', '.')
         cbtf_mc = '/sbin/cbtf_mrnet_commnode'
         cbtf_lmb = '/sbin/cbtf_libcbtf_mrnet_backend'
         run_env.set('XPLAT_RSH', 'ssh')
         run_env.set('MRNET_COMM_PATH',
                     join_path(self.spec['cbtf-krell'].prefix + cbtf_mc))
```

Comment:
@jgalarowicz FYI (I'll not hold up the PR for this, but thought you may want to know):

as it is now, `join_path` is getting a single argument (the `+` is resolved before the call) and isn't doing anything, so 

`run_env.set('MRNET_COMM_PATH', self.spec['cbtf-krell'].prefix + cbtf_mc)`

would be equivalent. I'm not sure if that would be guaranteed to work though (it depends on whether `.prefix` includes a `/`. I think you would want to do

`join_path(self.spec['cbtf-krell'].prefix, cbtf_mc)`